cranelift/interpreter/src/interpreter.rs

```diff
@@ -118,17 +118,29 @@ impl<'a> Interpreter<'a> {
                     maybe_inst = layout.first_inst(block)
                 }
                 ControlFlow::Call(called_function, arguments) => {
-                    let returned_arguments =
-                        self.call(called_function, &arguments)?.unwrap_return();
```

Comment:
Maybe remove the `unwrap_return` method to avoid this footgun for others?

Reply:
We use it a lot in the rest of the tests so it still has some use. But I'll add a `cfg(test)`

Reply:
The tests can wrap the expected return value in `ControlFlow::Return` I think if they avoid `unwrap_return`.

Reply:
`ControlFlow` doesen't implement `PartialEq` so we can't just do something like `assert_eq!(ControlFlow::Return(vec![...]), rets)`

It doesn't implement PartialEq because it has some function pointers in some of the arms.